curriculum/challenges/english/16-the-odin-project/top-introduction-to-flexbox/introduction-flexbox-question-j.md

```diff
@@ -0,0 +1,35 @@
+---
+id: 6571c34868e4b3b17d3957fb
+title: Introduction to Flexbox Question J
+challengeType: 15
+dashedName: introduction-flexbox-question-j
+---
+# --description--
+
+Search through [our handbook](https://www.freecodecamp.org/news/the-css-flexbox-handbook) and find the answer to the question below.
+
+# --question--
+
+## --text--
+
+Which properties are considered container elements?
```

Comment:
Is this question correct? It doesn't seem to line up with the answers.

Reply:
This should be properties not elements.

Reply:
Yeah the purpose of this question is which to use for a flex container element and which to use for flex item elements.

I need to word this one differently.